<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>605</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="verticalWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #eee;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QWidget" name="verticalWidget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Current database: </string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>5</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Run">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border-radius: 15px;
background-color: rgb(255, 255, 255);
}
QPushButton:hover {
    border-radius: 15px;
	border: 1px solid #fff;
    outline: none;
	background-color: #ececec;
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="dep/res.qrc">
              <normaloff>:/images/RunbtnPic.svg</normaloff>:/images/RunbtnPic.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Add">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border-radius: 15px;
background-color: rgb(255, 255, 255);
}
QPushButton:hover {
    border-radius: 15px;
	border: 1px solid #fff;
    outline: none;
	background-color: #ececec;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="dep/res.qrc">
              <normaloff>:/images/AddTab.svg</normaloff>:/images/AddTab.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar:item:selected {
background-color: rgba(127, 127, 127, 0.75); 
}</string>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="styleSheet">
     <string notr="true">QMenu {
background-color: rgba(255, 255, 255, 0.9); 
padding: 4px 0px;
border: 1px solid rgb(0, 0, 0);
}
QMenu:item {

}
QMenu:item:selected {
background-color: rgba(127, 127, 127, 0.75); 
color: rgb(255, 255, 255);
}

 

 </string>
    </property>
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="on_actionCreateTableTrig"/>
    <addaction name="actionOpen_database"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionStop"/>
    <addaction name="actionStart"/>
    <addaction name="actionBackup"/>
    <addaction name="actionRestore"/>
    <addaction name="actionDatabase_Info"/>
   </widget>
   <widget class="QMenu" name="menuQuery">
    <property name="styleSheet">
     <string notr="true">QMenu {
background-color: rgba(255, 255, 255, 0.9); 
padding: 4px 0px;
border: 1px solid rgb(0, 0, 0);
}
QMenu:item:selected {
background-color: rgba(127, 127, 127, 0.75); 
color: rgb(255, 255, 255);
}
 </string>
    </property>
    <property name="title">
     <string>Query</string>
    </property>
    <addaction name="actionNew_query"/>
    <addaction name="actionClose_query"/>
    <addaction name="actionRun_query"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="styleSheet">
     <string notr="true">QMenu {
background-color: rgba(255, 255, 255, 0.9); 
padding: 4px 0px;
border: 1px solid rgb(0, 0, 0);
}
QMenu:item:selected {
background-color: rgba(127, 127, 127, 0.75); 
color: rgb(255, 255, 255);
}
 </string>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionContacts"/>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuQuery"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="on_actionCreateTableTrig">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Create database</string>
   </property>
  </action>
  <action name="actionOpen_database">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open database</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionBackup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Backup</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="actionDatabase_Info">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Database Info</string>
   </property>
  </action>
  <action name="actionNew_query">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New query</string>
   </property>
  </action>
  <action name="actionClose_query">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close query</string>
   </property>
  </action>
  <action name="actionRun_query">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Run query</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionContacts">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Contacts</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="dep/res.qrc"/>
 </resources>
 <connections/>
</ui>
